sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -144,7 +152,206 @@ def plot(self, ax=None, *, name=None, **kwargs):
         self.figure_ = ax.figure
         return self
```

Comment:
I still find it counter-intuitive that plot returns self instead of ax, but that's unrelated to this PR

Reply:
I don't really recall if there is any reason. Maybe it allows pipelining some operations but it is true that pandas/seaborn usually return an axis